epub33/core/index.html

```diff
@@ -691,320 +744,646 @@ <h4>Conformance Checking</h4>
+								href="#attrdef-item-fallback"><code>fallback</code> attribute</a> that points to the
+							next possible resource for Reading Systems to try when they do not support its format.
+							Although not common, a fallback resource can specify another fallback, thereby making chains
+							many resources deep. The one requirement is that there must be at least one EPUB Content
+							Document in a fallback chain.</p>
+
+						<p>Although they are not directly listed in the spine, all of the resources in the fallback
+							chain are considered part of the spine, and by extension part of the spine plane, since any
+							may be used by a Reading System.</p>
+
+						<p>Refer to <a href="#sec-manifest-fallbacks"></a> for more information.</p>
+
+						<div class="caution" id="caution-fallbacks">
+							<p>Although manifest fallbacks fulfill the technical requirements of EPUB, there is little
+								practical support for them in Reading Systems. Their use is strongly discouraged as it
+								can lead to unreadable publications.</p>
+						</div>
+
+						<div class="note">
+							<p>It is possible to provide manifest fallbacks for EPUB Content Documents, but this is not
+								required or common. For example, a <a>Scripted Content Document</a> could have a
+								fallback to an unscripted alternative for Reading Systems that do not support
+								scripting.</p>
+						</div>
+					</section>
+
+					<section id="sec-content-plane">
+						<h5>The Content Plane</h5>
+
+						<p>The <dfn>content plane</dfn> classifies resources based on their use in rendering <a>EPUB
+								Content Documents</a> and <a>Foreign Content Documents</a>. These resources fall into
+							three categories: <a>Core Media Type Resources</a>, <a>Foreign Resources</a>, and <a>Exempt
+								Resources</a>.</p>
+
+						<p>A Core Media Type Resource is one that <a>Reading Systems</a> have to support, so it can be
+							used without restriction in EPUB or Foreign Content Documents. For more information about
+							Core Media Type Resources, refer to <a href="sec-core-media-types"></a>.</p>
+
+						<div class="note">
+							<p>Being a Core Media Type Resource does not mean that Reading Systems will always render
+								the resource, as not all Reading Systems support all features of EPUB 3. A Reading
+								System without a <a>Viewport</a>, for example, will not render visual content such as
+								images.</p>
+						</div>
+
+						<p>The opposite of Core Media Type Resources are Foreign Resources. These are resources that
+							Reading Systems are not guaranteed to support the rendering of. As a result, similar to how
+							using Foreign Content Documents in the spine requires fallbacks to ensure their rendering,
+							using Foreign Resources in content documents also requires fallbacks. These fallbacks are
+							provided in one of two ways: using the capabilities of the parent format or via manifest
+							fallbacks.</p>
```

Comment:
The term "host" format is used elsewhere (e.g., §2.2.3) instead of "parent format". I can live with any of the two (although I have a slight preference for "host") but we should be consistent.

Reply:
Ya, "parent" is kind of weird, as that would make the markup its... child? Changed to host.